scripts/build/build.sh

```diff
@@ -68,5 +73,12 @@ tar zcvf $pkg_name licenses conf mesher VERSION
 if [ $JOB_NAME != "" ]; then
     cp $release_dir/$pkg_name /var/lib/jenkins/mesher-release
 fi
+
+if [ $VERSION != "latest" ]; then
+    date=$(date +%Y-%m-%d)
+    DIR_NAME="mesher-release-$date"
+    mkdir -p /var/lib/jenkins/userContent/mesher-release/$DIR_NAME
+    cp $release_dir/$pkg_name /var/lib/jenkins/userContent/mesher-release/$DIR_NAME
+fi
 tar zcvf $WORK_DIR/mesher.tar.gz licenses conf start.sh mesher VERSION
```

Comment:
mesher.tar.gz need version in this name like mesher-1.4.12-linux-amd64.tar.gz

Reply:
$pkg_name will contain version also.  not just mesher (pkg_name="mesher-$VERSION-linux-amd64.tar.gz")